interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala

```diff
@@ -60,70 +61,54 @@ class SpelExpressionSpec extends FunSuite with Matchers {
+                                staticMethodInvocationsChecking: Boolean = defaultStaticMethodInvocationsChecking,
+                                methodExecutionForUnknownAllowed: Boolean = defaultMethodExecutionForUnknownAllowed,
+                                dynamicPropertyAccessAllowed: Boolean = defaultDynamicPropertyAccessAllowed): ValidatedNel[ExpressionParseError, TypedExpression] = {
     val validationCtx = ValidationContext(
       context.variables.mapValuesNow(Typed.fromInstance))
-    parse(expr, validationCtx, Map.empty, flavour, strictMethodsChecking = true, staticMethodInvocationsChecking = true, methodExecutionForUnknownAllowed = true,
-        dynamicPropertyAccessAllowed = true)
-  }
-
-  private def parseWithoutStrictMethodsChecking[T: TypeTag](expr: String, context: Context = ctx, flavour: Flavour = Standard): ValidatedNel[ExpressionParseError, TypedExpression] = {
-    val validationCtx = ValidationContext(context.variables.mapValuesNow(Typed.fromInstance))
-    parse(expr, validationCtx, Map.empty, flavour, strictMethodsChecking = false, staticMethodInvocationsChecking = true, methodExecutionForUnknownAllowed = false,
-      dynamicPropertyAccessAllowed = false)
-  }
-
-  private def parse[T: TypeTag](expr: String, context: Context = ctx, flavour: Flavour = Standard, staticMethodInvocationsChecking: Boolean = true, methodExecutionForUnknownAllowed: Boolean = false): ValidatedNel[ExpressionParseError, TypedExpression] = {
-    val validationCtx = ValidationContext(
-      context.variables.mapValuesNow(Typed.fromInstance))
-    parse(expr, validationCtx, Map.empty, flavour, strictMethodsChecking = true, staticMethodInvocationsChecking, methodExecutionForUnknownAllowed,
-      dynamicPropertyAccessAllowed = true)
-  }
-
-  private def parse[T: TypeTag](expr: String, validationCtx: ValidationContext): ValidatedNel[ExpressionParseError, TypedExpression] = {
-    parse(expr, validationCtx, Map.empty, Standard, strictMethodsChecking = true, staticMethodInvocationsChecking = true, methodExecutionForUnknownAllowed = false,
-      dynamicPropertyAccessAllowed = false)
-  }
-
-  private def parse[T: TypeTag](expr: String, validationCtx: ValidationContext, dictionaries: Map[String, DictDefinition],
-                                flavour: Flavour, strictMethodsChecking: Boolean, staticMethodInvocationsChecking: Boolean, methodExecutionForUnknownAllowed: Boolean,
-                                dynamicPropertyAccessAllowed: Boolean): ValidatedNel[ExpressionParseError, TypedExpression] = {
+    parseV(expr, validationCtx, dictionaries, flavour,
+      strictMethodsChecking = strictMethodsChecking,
+      staticMethodInvocationsChecking = staticMethodInvocationsChecking,
+      methodExecutionForUnknownAllowed = methodExecutionForUnknownAllowed,
+      dynamicPropertyAccessAllowed = dynamicPropertyAccessAllowed)
+  }
+
+  private def parseV[T: TypeTag](expr: String, validationCtx: ValidationContext,
```

Comment:
what does V stand for? ValidationContext?

Reply:
Yes. It can't be two overloaded parse methods with default values. I haven't got better idea how to handle it.

Reply:
ok, let's leave it